docs/Changelog.md

```diff
@@ -8873,3 +8873,76 @@ This version of the operator has been available since version 9 of the default O
 <dd>Constrain input and output types to float/int tensors.</dd>
 </dl>
 
+### <a name="Scatter-9"></a>**Scatter-9**</a>
+
+  Given `data`, `updates` and `indices` input tensors of rank r >= 1, write the values provided by `updates` 
+  into `data` along `axis` dimension of `data` (by default outer-most one as axis=0) at corresponding `indices`. 
+  For each entry in `updates`, the target index in `data` is specified by corresponding entry in `indices`
+  for dimension = axis, and index in source for dimension != axis. For instance, in a 2-D tensor case, 
+  data[indices[i][j]][j] = updates[i][j] if axis = 0, or data[i][indices[i][j]] = updates[i][j] if axis = 1.
```

Comment:
Something that was unclear to me is that `i` and `j` are not loop counters in terms of `data` or `output` but rather `indices` and `updates`. This greatly bewildered me when looking at example 2 because it appeared that "data[i][indices[i][j]] = updates[i][j] if axis = 1" would yield out of bounds array accesses. Propose clarifying that...

```suggestion
  data[indices[i][j]][j] = updates[i][j] if axis = 0, or data[i][indices[i][j]] = updates[i][j] if axis = 1, where i and j are loop counters from 0 up to the respective size in `updates` - 1.
```